alas_but_one.py

```diff
@@ -17,6 +17,8 @@
 # of the type. Hence, "there was, alas, but one".
 
 MAX_OCCUR=1
+OUTPUT_FILE='alas_output.txt'
+EXCLUSION_FILE="alas_exclude.txt"
```

Comment:
The issue with an example exclusion file is that it is going to vary widely from one document set to another. For example, a doc set might have a single use of "Alas" which is valid. And while we may want it on the exclusion list for *that* driver, we don't want to automatically include it in all drivers since in other drivers it may be a typo for "Atlas". The exclusion lists are going to be extremely driver-dependent, and not a running dictionary of words we want to ignore.

Reply:
> The exclusion lists are going to be extremely driver-dependent, and not a running dictionary of words we want to ignore.

Then it seems like it would make sense to have one per driver. I think a sample file named "java-sync-exclusion-list.txt" would help communicate the intent and value. You can add this in the PR to the new directory: https://github.com/ccho-mongodb/miscellaneous-tools/tree/master/alas_but_one